snowdrop-execution/src/Snowdrop/Execution/Expand.hs

```diff
@@ -67,27 +70,26 @@ runSeqExpandersSequentially
     ( HasBException conf CSMappendException
     , HasLens (Ctx conf) (ChgAccumCtx conf)
     , MappendHChSet (UnionSeqExpandersInps txtypes)
-
     , Default (ChgAccum conf)
-    , Default (SumChangeSet (UnionSeqExpandersInps txtypes))
+    , Default (HChangeSet (UnionSeqExpandersInps txtypes))
     )
     => Rec (ProofNExp conf) txtypes
     -> [SomeData TxRaw (Both (ExpandableTx txtypes) c)]
     -> ERoCompM conf (UnionSeqExpandersInps txtypes) (OldestFirst [] (SomeTx c, ChgAccum conf))
 runSeqExpandersSequentially proofNExps allTxs =  do
-      someTxs :: [(SomeTx c, HChangeSet (UnionSeqExpandersInps txtypes))] <- foldM runExps [] allTxs
+      (someTxs, _) <- foldM runExps ([], def) allTxs
       let (fch, hch) = unzip someTxs
       modAccums :: OldestFirst [] (ChgAccum conf) <- modifyAccum (OldestFirst hch)
       pure $ OldestFirst (zip fch (unOldestFirst modAccums))
 
   where
     runExps
-        :: [(SomeTx c, HChangeSet (UnionSeqExpandersInps txtypes))]
+        :: ([(SomeTx c, HChangeSet (UnionSeqExpandersInps txtypes))], HChangeSet (UnionSeqExpandersInps txtypes))
```

Comment:
I think that without `HChangeSet (UnionSeqExpandersInps txtypes)` we should actually pass and updated `chgAccum`.

This way `modAccums :: OldestFirst [] (ChgAccum conf) <- modifyAccum (OldestFirst hch)` won't be needed at all